make/test.mk

```diff
@@ -153,8 +153,9 @@ endif
 
 .PHONY: clean-e2e-resources
 clean-e2e-resources:
-	oc get catalogsource --output=name -n openshift-marketplace | grep "toolchain-operator" | xargs oc delete -n openshift-marketplace 2>/dev/null || true
-	oc get subscription --output=name -n ${TOOLCHAIN_NS} |  grep "toolchain-operator" | xargs oc delete -n ${TOOLCHAIN_NS} 2>/dev/null || true
-	oc get subscription --output=name -n openshift-operators |  grep "openshift-pipelines-operator" | xargs oc delete -n openshift-operators 2>/dev/null || true
-	oc get csv --output=name -n openshift-operators |  grep "codeready-toolchain-operator" | xargs oc delete -n openshift-operators 2>/dev/null || true
+	-oc get catalogsource --output=name -n openshift-marketplace | grep "toolchain-operator" | xargs oc delete -n openshift-marketplace 2>/dev/null || true
+	-oc get subscription --output=name -n ${TOOLCHAIN_NS} | grep "toolchain-operator" | xargs oc delete -n ${TOOLCHAIN_NS} 2>/dev/null || true
+	-oc get subscription --output=name -n openshift-operators | grep "openshift-pipelines-operator" | xargs oc delete -n openshift-operators 2>/dev/null || true
+	-oc get csv --output=name -n openshift-operators | grep "codeready-toolchain-operator" | xargs oc delete -n openshift-operators 2>/dev/null || true
+	-oc get clusterrole --output=name  | grep "codeready-toolchain-operator" | xargs oc delete 2>/dev/null || true
```

Comment:
right, I should probably remove the `|| true` as well, since the `-` prefix is an idiom of Makefile.

Reply:
well, I added the suffix `|| true` because the output looks better, so I would keep the suffix and remove the prefix.
Or - after all - you can keep them both... :-)

Reply:
well, the `-` prefix is a feature of `Makefile` so I would rather keep it. But we can have both, indeed. So I'll just keep the change as-is for now, then ;)

Reply:
ok, agree - sorry for this unnecessary discussion :-D

Reply:
no worries ;)